core/ibft.go

```diff
@@ -1006,7 +1005,6 @@ func (i *IBFT) buildProposal(ctx context.Context, view *proto.View) *proto.Messa
 func (i *IBFT) acceptProposal(proposalMessage *proto.Message) {
```

Comment:
Change the comment of this function not to include `moves the state`

Reply:
Fixed 94978a3b28f7a8bef95d9fe77e78fea524c6c1eb